src/Microsoft.PowerShell.Commands.Management/commands/management/TestConnectionCommand.cs

```diff
@@ -985,8 +985,8 @@ public class PingMtuStatus : PingStatus
             /// <param name="destination">The destination machine name of the ping.</param>
             /// <param name="reply">The response from the ping attempt.</param>
             /// <param name="bufferSize">The buffer size from the successful ping attempt.</param>
-            internal PingMtuStatus(string source, string destination, PingReply reply, int bufferSize)
-                : base(source, destination, reply, 1)
+            internal PingMtuStatus(string source, string destination, PingReply? reply, int bufferSize)
+                : base(source, destination, reply ?? throw new SystemException(nameof(reply)), 1)
```

Comment:
@xtqqczze Why do you mess style and functional changes in one PR?
If you think we should change exception types you should do this in separate PR. (Although I see no point in it). Please fix only exception arguments in the PR.

Reply:
Changes to exception types are required to enable CA2208, if `ArgumentException` is the wrong type. The exception type must change if the `paramName` parameter cannot be made to match the name of a parameter.